torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -323,6 +363,16 @@ script::Module optimizeForMobile(
 
 #else
 
+void transformConv1dToConv2d(std::shared_ptr<Graph>& graph) {
+  TORCH_INTERNAL_ASSERT(
+      "XNNPACK is not enabled. Please build with USE_XNNPACK=1");
+}
+
+void transformConv1dToConv2d(script::Module& module) {
+  TORCH_INTERNAL_ASSERT(
+      "XNNPACK is not enabled. Please build with USE_XNNPACK=1");
+}
+
```

Comment:
Actually, this is just rewriting graph into another graph that does not need xnnpack support to be executed. So we can remove these and move the other functions you added outside of ifdef.
Eventually we can think of moving them to a separate file once we have these working.